### PR TITLE
Update store document to match 2.0

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun @cvasilak Could you review that?

File: docs/guides/aerogear-android/store.asciidoc [hunk 1]
```diff
@@ -3,42 +3,23 @@ layout: basic
...
+To create a link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/datamanager/Store.html[*Store*], invoke  link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/DataManager.html[*DataManager*] config method, passing along a link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/impl/datamanager/StoreConfiguration.html[*StoreConfiguration*]. link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/impl/datamanager/MemoryStoreConfiguration.html[*MemoryStoreConfiguration*] for an in-memory backed store, or link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/impl/datamanager/SQLStoreConfiguration.html[*SQLStoreConfiguration*] for an sqlite backed store. A name should be also provided to distinquish between different stores that this link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/DataManager.html[*DataManager*] can hold.
```
Comment (cvasilak) [inline docs/guides/aerogear-android/store.asciidoc]: s/distinquish/distinguish

File: docs/guides/aerogear-android/store.asciidoc [hunk 2]
```diff
@@ -124,16 +111,15 @@ Take a look at the complete example in our link:https://github.com/aerogear/aero
 
 ==== Encrypted Stores
 
-If your storage information need to be encrypted, AeroGear provides convenient and transparent encrypted storage support, building on the security foundation provided by link:https://github.com/aerogear/aerogear-crypto-java[aerogear-crypto-java], encrypted variants of both memory (StoreTypes.ENCRYPTED_MEMORY) and SQLite (StoreTypes.ENCRYPTED_SQL) are provided.
+If your storage information need to be encrypted, AeroGear provides convenient and transparent encrypted storage support, building on the security foundation provided by link:https://github.com/aerogear/aerogear-crypto-java[aerogear-crypto-java], encrypted variants of both memory (link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/impl/datamanager/EncryptedMemoryStore.html[*EncryptedMemoryStore*]) and SQLite (link:/docs/specs/aerogear-android-store/org/jboss/aerogear/android/impl/datamanager/EncryptedSQLStore.html[*EncryptedSQLStore*])
```
Comment (cvasilak) [inline docs/guides/aerogear-android/store.asciidoc]: s/need/needs

**Comment by cvasilak** [comment]
apart from the small typos, looks good :+1:

**Comment by danielpassos** [comment]
@cvasilak Done. Thanks!

**Comment by secondsun** [comment]
Landed @ 839911ad867cacf5c2a3d1786321bb1a006333f5

**Pull request merged**